manifests/init.pp

```diff
@@ -140,6 +140,10 @@
 #                                           Defaults to undef. If 'windows' or 'msi' are
 #                                           used as the provider then this setting is
 #                                           required.
+# $package_install_options::                Flags that should be passed to the installer
+#                                           during installation. Defaults to undef. May be
+#                                           a string or a hash, see Puppet Package resource
+#                                           documentation for more information
```

Comment:
```suggestion
# $package_install_options::                Flags that should be passed to the package manager
#                                           during installation. Defaults to undef. May be
#                                           a string or a hash, see Puppet Package resource
#                                           documentation for the provider matching your package manager
```

I think this is a little clearer for users. In practice these modules are only supported on a few OSes so the actual number of package providers we support is small. Users would more quickly get an idea of what this does by going straight to the documentation for the `yum` Package provider or equivalent